Azure.ResourceManager.Core/ResourceContainerBase.cs

```diff
@@ -8,77 +8,80 @@
 namespace Azure.ResourceManager.Core
 {
     /// <summary>
-    ///     Create known Container and Leaf scopes for ARM Containers
+    /// Base class for resource container
+    /// TODO:    Create known Container and Leaf scopes for ARM Containers
     ///     Think about how to extend known scope types in an extensible fashion (is it just adding them to the default, or is
```

Comment:
Can we remove this TODO?  If its still an open question lets add it to the agenda for the .net sync and create action items based on the outcome of the discussion